tests/system/fim/common_tasks/delete_files.py

```diff
@@ -13,7 +13,27 @@
 import os
 import time
 
-def delete_files(input_file_path, n, output_file_path, bunch_size=100, wait_time=100):
+
+def delete_file(path, attempt=0, sleep_time=5.0):
+    success = True
+    try:
+        if attempt > 0:
+          time.sleep(sleep_time)
+          logging.info(f"Failed to delete {path}, retry: {attempt}")
+        if attempt > 10:
+          return False
+        open(path, 'a').close() # (To update mtime_after)
+        os.remove(path)
+    except FileNotFoundError as e:
+        logging.error("File " + path + " not found.", exc_info=True)
+        raise e
+    except PermissionError:
+        logging.info("File " + path + " used by another process.", exc_info=True)
+        success = delete_file(path, attempt+1)
+        pass
+    return success
+
+def delete_files(input_file_path, n, output_file_path, bunch_size=500, wait_time=0, rt_delay=0):
```

Comment:
Please update default values to 90 `bunch_size` and 1 for `rt_delay`

Reply:
Okay, I set it to 500 and 0 because maybe in some test it is not required to have such delay.

Reply:
https://github.com/wazuh/wazuh-qa/pull/570/commits/d4b083a81d9e4af48ecc79f1f08e4177359c9226